### DISCUSSION TITLE
MOVE项目发起 MoveAS(Algorithmic Stablecoins)

### DISCUSSION THREAD

**Discussion by mason0510** [NonceGeek/Move-Camp#226] [Amazing Things!]
**一句话idea**
一个改善版的Move生态的AS算法稳定币项目
我的角色: 后台开发者 合约开发者
人数需求：前端仍缺1人 美工1人 运营1人 
我的联系方式: Wechat-组队阿特我-随时都在
个人简介：一只有梦想的猫

**Comment by Demian101** [comment]
打算基于 Sui 嘛

**Reply by mason0510** [reply]
是的。

**Comment by mason0510** [comment]
完善介绍：
此项目的灵感来源于自己经常参与的稳定币项目UST和另一个AMPL类的项目IMMO。通过将博弈论和质押绑定引入市场，实现一揽子的去中心化的货币储备协议。 每一个代币的价值都是USDT等稳定币的内在价值支撑。针对多个算法稳定币乌托邦式的做法以及羊毛党，设计了独特的动态调整机制，通过通胀和通缩机制，独特的债券发行机制，实现超额收益的同事，实现风险分散，维持整个系统的稳定运行。希望能够实现在sui生态中发挥独特的价值。

目标
成为sui生态中广泛使用的真正的超额抵押的稳定币

计划：
1.前期，实现简单版的算法稳定币的发行。 前端实现简单页面展示，能够质押和提取即可。
2.中期，实现平台的兑换功能。 能够在生态内将稳定币兑换成其他货币。
3.后期，实现在去中心化游戏的应用。 能够给稳定币持续赋能。

第一版实现UI展示和发行即可。 重点工作是前端开发，合约开发和测试。 另外加上一部分的后台数据统计工作。

需要的帮助：对sui move 感兴趣。 能够积极完成任务。

**Comment by xiegudong45** [comment]
想问下现在还缺人不？

**Reply by mason0510** [reply]
> 想问下现在还缺人不？

缺人 你还在吗 加我 给我留言微信号 我加你

**Reply by xiegudong45** [reply]
谢谢，不过我已经加入其他项目了

**Comment by mason0510** [comment]
仓库地址:https://github.com/mason0510/react-sui.git
合约地址：https://github.com/mason0510/suiAsContracts.git

**Comment by mason0510** [comment]
https://github.com/mason0510/mason-sui.git

**Reply by mason0510** [reply]
把前端和合约整到了一起。
